apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/api/client/BftScanConnection.scala

```diff
@@ -978,13 +979,31 @@ object BftScanConnection {
           (newScansFailedConnections, newScansSuccessfulConnections) <- attemptConnections(
             newScans
           )
+
+          newState = BftState(
+            (currentScanConnections -- removedScans) ++ newScansSuccessfulConnections,
+            (currentFailed -- removedScans) ++ newScansFailedConnections,
+          )
+
+          successfullyConnectedUrls = newState.openConnections.keys.toSet
+
+          filteredScans: Seq[DsoScan] = scansInDsoRules
+            .filter(scan => successfullyConnectedUrls.contains(scan.publicUrl))
+
+          dsoScanSeq: Seq[(String, String)] = filteredScans.map(scan =>
+            (scan.svName, scan.publicUrl.toString)
+          )
+
+          _ = persistScanUrlsCallback.map(f => f(dsoScanSeq))
```

Comment:
Did we explicitly chose to persist scans that we managed to connect to instead of all the scans? I would honestly expect to have all the scans persisted, even if we can't connect to some of them